### PR TITLE
Add explanatory comments on Rx/MVI dense code.

### LINKED ISSUE #11: Add comments everywhere
- [ ] add a comment to every rx statement/chain
- [ ] add in readme why I don't use flowable

### PR DESCRIPTION
**Pull request by oldergod**
The PR aims to add enough comments to the base code for someone not familiar with [RxJava](https://github.com/ReactiveX/RxJava) or the MVI architecture to understand what the code does where any of them is used.

fixes #11

### CODE REVIEW

**Comment by oldergod** [comment]
@malmstein @JoseAlcerreca If you have time to see if it's going in the right direction, please do.

File: README.md [hunk 1]
```diff
@@ -71,7 +71,8 @@ The _State_ contains all the information the _View_ needs to render itself.
 
 ### Observable
 
-[RxJava2](https://github.com/ReactiveX/RxJava) is used in this sample. The data model layer exposes RxJava `Observable` streams as a way of retrieving tasks. In addition, when needed, `void` returning setter methods expose RxJava `Completable` streams to allow composition inside the _ViewModel_.
+[RxJava2](https://github.com/ReactiveX/RxJava) is used in this sample. The data model layer exposes RxJava `Observable` streams as a way of retrieving tasks. In addition, when needed, `void` returning setter methods expose RxJava `Completable` streams to allow composition inside the _ViewModel_.  
+`Observable` is used over `Flowable` streams for backpressure is not (and doesn't need to be in this project) handled. Using `Observable` clearly convey that backpressure is not a concern.
```
Comment (JoseAlcerreca) [inline README.md:75]: Use simpler English, "because backpressure" or "as backpressure"

Comment (JoseAlcerreca) [reply README.md:75]: Using `Observable` clearly convey that backpressure is not a concern. -> rm

**Review by JoseAlcerreca — COMMENTED** [review]
Awesome job!

File: README.md (hunk 1 above)
Comment (oldergod) [reply README.md:75]: thank you, done!

**Pull request merged**